### DISCUSSION TITLE
Delete invite

### DISCUSSION THREAD

**Discussion by redstonepro109** [discord-jda/JDA#2058] [Questions and Help] [answered]
How to delete a invite that was created

**Comment by Tais993** [comment, marked as answer]
You first use Invite#resolve, to resolve the invite.
After that, a flatmap where you call the `delete` method on Invite
```java
Invite.resolve(jda, inviteCode).flatmap(Invite::delete).queue();
```
Something like this should work

**Reply by redstonepro109** [reply]
Thank you very much !

**Reply by redstonepro109** [reply]
Do you know the format of the inviteCode? Like https://discord.gg/invitecode or just the invite code (ex. JcIvsHs)

**Reply by Tais993** [reply]
The invite code itself
